examples/basic/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "hermes-example-basic",
+  "version": "1.0.0",
+  "description": "A basic usage example for hermes",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "StreamMe",
+  "license": "ISC",
+  "dependencies": {
+    "@streammedev/flux-store": "^1.0.0-BETA-2",
+    "@streammedev/hermes": "file:../../",
+    "babelify": "^7.3.0",
```

Comment:
`npm i` -> `Failed at the @streammedev/hermes@0.0.1 babel script 'babel src -d lib'.`